bee-ledger/src/types/receipt.rs

```diff
@@ -41,7 +41,7 @@ impl Receipt {
             migrated_amount =
                 migrated_amount
                     .checked_add(funds.output().amount())
-                    .ok_or(Error::InvalidMigratedFundsAmount(
+                    .ok_or_else(|| Error::InvalidMigratedFundsAmount(
```

Comment:
Why ?

Reply:
There was a clippy warning, should I undo it?

Reply:
Ah alright, weird. No no don't undo, thanks.